### PR TITLE
forbid all explicit str concat Fixes #6

### LINKED ISSUE #6: Extend to explicit string literal concatenation? [Enhancement]
You might consider this out of scope (given the current name of the plugin is short for "flake8 implicit string concatenation"), but I found it in the context of running black on legacy code.

Simplified example:

```python
# Common to get this kind of thing after running black
# where long strings had been split over multiple lines
# with explicit concatenation - style wise this is bad:
msg = "Hello " + "world"
print(msg)
```

Here I'd like a style warning, and would prefer this:

```python
msg = "Hello world"
print(msg)
```

Real example from https://github.com/biopython/biopython/pull/2359 code snippet https://github.com/biopython/biopython/blob/biopython-175/BioSQL/BioSeqDatabase.py#L772

```python
        sql = "DELETE FROM bioentry " + \
              "WHERE biodatabase_id=%s AND bioentry_id=%s;"
```

After black we have:

```python
        sql = "DELETE FROM bioentry " + "WHERE biodatabase_id=%s AND bioentry_id=%s;"
```

I want a flake8 warning, so we can spot this automatically and fix it:

```python
        sql = "DELETE FROM bioentry WHERE biodatabase_id=%s AND bioentry_id=%s;"
```

Both the before and after explicit string literals could trigger a flake8 violation?

### PR DESCRIPTION
**Pull request by keisheiled**

### CODE REVIEW

File: flake8_implicit_str_concat.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 """
 Forbid implicitly concatenated string literals on one line such as those
-introduced by Black
+introduced by Black.
+Forbid all explicitly concatenated strings.
 """
```
Comment (hugovk) [inline flake8_implicit_str_concat.py]: At https://pypi.org/project/flake8_implicit_str_concat/ the one-line summary is cut short:

image

https://github.com/takluyver/flit#usage says

> Make sure that foobar's docstring starts with a one-line summary of what the module is

So perhaps add a short one-liner on the first line?

Comment (keisheiled) [reply flake8_implicit_str_concat.py]: @hugovk https://github.com/keisheiled/flake8-implicit-str-concat/issues/11

**Comment by keisheiled** [comment]
this also forbids any of these aberrant constructs:

```python
ham = (
    "spam"
    + "eggs"
)

ham = "spam" \
      + "eggs"
```

**Comment by peterjc** [comment]
Looks good in my local testing - found a couple I had missed on earlier version of this pull request:
https://github.com/biopython/biopython/pull/2359

**Pull request merged**